packages/stream_chat_flutter_core/lib/src/channels_bloc.dart

```diff
@@ -148,7 +150,13 @@ class ChannelsBlocState extends State<ChannelsBloc>
     final client = StreamChatCore.of(context).client;
 
     if (!widget.lockChannelsOrder) {
-      _subscriptions.add(client.on(EventType.messageNew).listen((e) {
+      _subscriptions.add(client
+          .on(
+        EventType.messageNew,
+        EventType.notificationAddedToChannel,
+        EventType.notificationMessageNew,
```

Comment:
adding additional events are enough to solve this issue.

Reply:
We're already handling this event in ChannelListView, but the data is not being loaded because the pagination is ended.
This is the issue, and that happens also for the other events in that handler